B/basiclu/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "basiclu"
+version = v"2.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ERGO-Code/basiclu.git", "5c882bd2617006d0d1ee6ba3400686da092ecd85"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+if [[ ${target} == *mingw32* ]]; then
+    for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+        atomic_patch -p1 ${f}
+    done
+fi
+cd basiclu/
+if [[ ${target} == *mingw32* ]]; then
+    make CC99="cc -std=c99"
+elif [[ ${target} == *-apple-* ]] || [[ ${target} == *freebsd* ]]; then
+    make CC99="cc -std=c99" -D_DARWIN_C_SOURCE
+else
+    make CC99="cc -std=c99" LDLIBS="-lm -lrt"
+fi
```

Comment:
I'd simplify to
```suggestion
if [[ "${target}" == *-apple-* ]] || [[ "${target}" == *freebsd* ]]; then
    CFLAGS="-D_DARWIN_C_SOURCE"
elif [[ "${target}" == *-linux-* ]]; then
    LDLIBS="-lm -lrt"
fi
make CC99="cc -std=c99" CFLAGS="${CFLAGS}" LDLIBS="${LDLIBS}"
```
but I'm not sure `-D_DARWIN_C_SOURCE` with FreeBSD is a good idea

Reply:
Thanks !
For FreeBSD,
```
grep -r -B2 '#define CLOCK_MONOTONIC_RAW\b' /opt/${target}/${target}
```
returns nothing :(

Reply:
Conclusion : Let's remove FreeBSD platform.

Reply:
Right, so maybe FreeBSD isn't really supported and you can skip it.  One option could be to define `CLOCK_MONOTONIC_RAW` as `CLOCK_MONOTONIC` (`-DCLOCK_MONOTONIC_RAW=CLOCK_MONOTONIC`), but I don't know what the consequences would be, I have no clue what this stuff does.

Reply:
If didn't work, I remove FreeBSD platform :)